shopfloor_reception/migrations/14.0.2.1.0/pre-migration.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+import logging
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from openupgradelib import openupgrade
+except (ImportError, IOError) as err:
+    _logger.debug(err)
```

Comment:
Why `try`. If it cannot be imported, the migration will fail. Why not adding it in the repo requirements.txt ?

Reply:
Agree. Removed the try and added the dependency.